### DISCUSSION TITLE
What is the largest citrus fruit?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#472] [Galaxy Trivia] [answered]
**Bonus:** What is the smallest citrus fruit?

**Comment by ParanoidUser** [comment]
@aliabdallah7, @ralphcajipe, @dualsgo, any ideas? 🍋

**Reply by ralphcajipe** [reply]
The largest citrus fruit is the pomelo, which can grow to be up to 12 inches in diameter and weigh up to 4 pounds. The smallest citrus fruit is the kumquat, which is about the size of an egg and typically grows to be 2-3 inches in diameter.

**Comment by aliabdallah7** [comment, marked as answer]
The largest citrus fruit in the world is the (pomelo).

**Reply by ParanoidUser** [reply]
Well done! 👏👏 Congratulations on your well deserved [Galaxy Brain](https://github.com/aliabdallah7?tab=achievements&achievement=galaxy-brain) badge. Continue to unlock the next badge levels (Bronze for 8 answers), and don't hesitate to post your own questions to the community. 😊🎉
